### PR TITLE
[.NET] Parse TextBlock style

### LINKED ISSUE #5478: [.NET] Parser support for TextBlock headings

### LINKED ISSUE #5479: [WPF] Renderer support for TextBlock headings
Narrator supports navigation by headings. See [support article](https://support.microsoft.com/en-us/windows/chapter-5-navigation-68941680-3245-6ef5-5012-0674b8b6fc59) that describes how to cycle through navigation modes.

Text with `style: heading` receive the appropriate accessibility flags on other platforms to support this mode of navigation, including UWP XAML. However, the .NET WPF renderer doesn't seem to offer this support (see details in #5626).

Consult with accessibility team to determine if there is a workaround, or close this as external.

### PR DESCRIPTION
**Pull request by golddove**
## Related Issue
Fixes #5478 

## Description
.NET WPF doesn't support HeadingLevel on AutomationProperties (see: https://github.com/dotnet/wpf/issues/1825), unlike UWP XAML. So, this only has object model / host config parsing, if the host wants to do something with the property (and for future heading styling).

## How Verified
Added tests

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5626)

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards/Rendering/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (RebeccaAnne) [inline source/dotnet/Library/AdaptiveCards/Rendering/Class1.cs:1]: Can we give this file a more descriptive name? (Probably HeadingsConfig.cs)

**Comment by RebeccaAnne** [comment]
> - [ActionsConfig](#T-AdaptiveCards-Rendering-ActionsConfig 'AdaptiveCards.Rendering.ActionsConfig')

I think this file should be gitignored.

---
Refers to: source/dotnet/Library/AdaptiveCards/docs/AdaptiveCards.md:6 in 6ba267a. [](commit_id = 6ba267ade9c09152ecddfc7e5adb02a4341d4d01, deletion_comment = False)

**Comment by golddove** [comment]
@RebeccaAnne gitignore and remove the files from the repo? I think that should be a separate PR.

**Comment by golddove** [comment]
I've removed the changes from this PR for now, but we should either remove the file entirely, or keep it updated.

**Review by RebeccaAnne — APPROVED** [review]
:shipit:

**Pull request merged**